tests/functional/interfaces-with-arguments.ts

```diff
@@ -0,0 +1,105 @@
+import "reflect-metadata";
+import { getMetadataStorage } from "../../src/metadata/getMetadataStorage";
+import {
+  IntrospectionSchema,
+  IntrospectionObjectType,
+  IntrospectionInterfaceType,
+  IntrospectionField,
+} from "graphql";
+import { getSchemaInfo } from "../helpers/getSchemaInfo";
+import { ArgsType, Field, Int, InterfaceType, ID, Query, ObjectType, Arg, Args } from "../../src";
+
+describe("Interfaces with arguments", () => {
+  describe("Schema", () => {
+    let schemaIntrospection: IntrospectionSchema;
+    let queryType: IntrospectionObjectType;
+
+    beforeAll(async () => {
+      getMetadataStorage().clear();
+
+      @ArgsType()
+      class ImageArgs {
+        @Field(type => Int, { nullable: true })
+        width?: number;
+        @Field(type => Int, { nullable: true })
+        height?: number;
+      }
+
+      @InterfaceType()
+      abstract class Article {
+        @Field(type => ID)
+        id: string;
+        @Field({ nullable: true })
+        headline?: string;
+
+        @Field()
+        image1(@Arg("width") width: number, @Arg("height") height: number): string {
+          return `http://lorempixel.com/${width}/${height}/`;
+        }
+        @Field()
+        image2(@Args(type => ImageArgs) args: any): string {
+          return `http://lorempixel.com/${args.width}/${args.height}/`;
+        }
+      }
+
+      @ObjectType({ implements: Article })
+      class WebArticle extends Article {
+        @Field()
+        url: string;
+
+        @Field()
+        image3(@Arg("width") width: number, @Arg("height") height: number): string {
```

Comment:
Please use more meaningfull testing names - `interfaceFieldMethod` and `implemetingObjectTypeFieldMethod`, just like in other test suites

Reply:
okay, done :)